protos/feast/core/CoreService.proto

```diff
@@ -102,9 +105,6 @@ message GetFeatureSetRequest {
 
     // Name of feature set (required).
     string name = 1;
-
-    // Version of feature set (optional). If omitted then latest feature set will be returned.
-    int32 version = 2;
```

Comment:
Should we add a reserved keyword here?

https://developers.google.com/protocol-buffers/docs/proto#reserved

Reply:
donezo

Reply:
Is it? Not seeing it for this or `feature_set_version` on `ListFeatureSetsRequest`